syncer/dml.go

```diff
@@ -569,9 +568,9 @@ func (s *Syncer) mappingDML(schema, table string, ti *model.TableInfo, data [][]
 		rows = make([][]interface{}, len(data))
 	)
 	for i := range data {
-		rows[i], _, err = s.columnMapping.HandleRowValue(schema, table, columns, data[i])
+		rows[i], _, err = s.columnMapping.HandleRowValue(table.Schema, table.Name, columns, data[i])
```

Comment:
Why not also change `HandleRowValue` to accept a `filter.Table`

Reply:
It belongs to `tidb-tools`.

Reply:
😂ok.